src/Datadog.Trace/TraceId.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// Class representing 64 or 128 bit TraceID.
+    /// </summary>
+    public readonly struct TraceId
```

Comment:
I think you might have forgotten to add `: IEquatable<TraceId>`

Reply:
Added

Reply:
Resolve